UnitTests/GitUITests/CommandsDialogs/ReferenceRepository.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using CommonTestUtils;
+using GitCommands;
+
+namespace GitUITests.CommandsDialogs
+{
+    public class ReferenceRepository : IDisposable
```

Comment:
Was the intention for this class to have a destructor?

Reply:
No

Reply:
Why the finalizer/disposable pattern here then?

Reply:
This pattern applies to any class which is not `sealed`.

Reply:
Subclasses can override `Dispose` and still add this logic if they have unmanaged resources. Am I missing something, or is this a matter of preference?